### DISCUSSION TITLE
RHEL 7 and SELinux

### DISCUSSION THREAD

**Discussion by atbohmer** [phaag/nfdump#348] [Q&A]
Hello,

Any experience or howto available about getting SELinux and nfdump not biting each other?
Runninng nfdump as a daemon goes fine, but when called from within nfsen/nfsen-ng to create a flow report or such it fails:
SELinux is preventing nfdump from getattr access on the file /nfdump/profiles-data/live/...

TIA,
Andre

**Comment by phaag** [comment]
Closed off topic.
